bundles/org.openhab.binding.homeconnect/README.md

```diff
@@ -135,7 +135,7 @@ The Home Connect bridge can be configured in the openHAB UI as follows:
 
 ## Examples: File based configuration
 
-If you prefer to configure everything via file instead of openHAB UI, here are some examples.
+If you prefer to configure everything via file instead of openHAB UI, here are some examples. In order to get your `haId` you might use the discover feature in paper UI, copy the appliance's ID and paste it into your `*.things`.
```

Comment:
I would be easier to view differences if there is only once sentence per line.
```suggestion
If you prefer to configure everything via file instead of openHAB UI, here are some examples.
In order to get your `haId` you might use the discover feature in paper UI, copy the appliance's ID and paste it into your `*.things`.
```